universal_transfer_operator/src/universal_transfer_operator/data_providers/filesystem/aws/s3.py

```diff
@@ -123,41 +125,48 @@ def download_file(self, file) -> TempFile:
 
     @property
     def verify(self) -> str | bool | None:
-        return self.dataset.extra.get("verify", None)
+        verify: str | bool | None = self.dataset.extra.get("verify", None)
+        return verify
 
     @property
     def transfer_config_args(self) -> dict | None:
-        return self.dataset.extra.get("transfer_config_args", {})
+        transfer_config_args: dict | None = self.dataset.extra.get("transfer_config_args", {})
+        return transfer_config_args
 
     @property
     def s3_extra_args(self) -> dict | None:
-        return self.dataset.extra.get("s3_extra_args", {})
+        s3_extra_args: dict | None = self.dataset.extra.get("s3_extra_args", {})
+        return s3_extra_args
 
     @property
     def bucket_name(self) -> str:
-        bucket_name, _ = self.hook.parse_s3_url(self.dataset.path)
-        return bucket_name
+        url: tuple[str, str] = self.hook.parse_s3_url(self.dataset.path)
+        return url[0]
 
     @property
     def s3_key(self) -> str:
-        _, key = self.hook.parse_s3_url(self.dataset.path)
-        return key
+        url: tuple[str, str] = self.hook.parse_s3_url(self.dataset.path)
+        return url[1]
 
     @property
     def s3_acl_policy(self) -> str | None:
-        return self.dataset.extra.get("s3_acl_policy", None)
+        s3_acl_policy: str | None = self.dataset.extra.get("s3_acl_policy", None)
```

Comment:
```suggestion
        s3_acl_policy: str | None = self.dataset.extra.get("s3_acl_policy")
```
The default behaviour of `get` is already to return `None`